From 2bfcdab2c3a74a0be008264283ccbc429e5b347c Mon Sep 17 00:00:00 2001
From: "Jan Chren (rindeal)" <dev.rindeal@gmail.com>
Date: Fri, 24 Mar 2017 15:30:16 +0100
Subject: [PATCH] Improve htop.desktop file

- sort entries according to the spec
- add to the `Monitor` category
---
 htop.desktop | 66 ++++++++++++++++++++++++++++++------------------------------
 1 file changed, 33 insertions(+), 33 deletions(-)

diff --git a/htop.desktop b/htop.desktop
index 13b71e79..1ae5d60f 100644
--- a/htop.desktop
+++ b/htop.desktop
@@ -1,7 +1,35 @@
 [Desktop Entry]
+Type=Application
 Version=1.0
 Name=Htop
-Type=Application
+GenericName=Process Viewer
+GenericName[ca]=Visor de processos
+GenericName[de]=Prozessanzeige
+GenericName[en_GB]=Process Viewer
+GenericName[es]=Visor de procesos
+GenericName[fi]=Prosessikatselin
+GenericName[fr]=Visualiseur de processus
+GenericName[gl]=Visor de procesos
+GenericName[it]=Visore dei processi
+GenericName[ko]=프로세스 뷰어
+GenericName[nb]=Prosessviser
+GenericName[nl]=Viewer van processen
+GenericName[nn]=Prosessvisar
+GenericName[pl]=Przeglądarka procesów
+GenericName[pt]=Visualizador de Processos
+GenericName[pt_BR]=Visualizador de processos
+GenericName[ru]=Монитор процессов
+GenericName[sk]=Prehliadač procesov
+GenericName[sl]=Pregledovalnik opravil
+GenericName[sr@ijekavian]=Приказивач процеса
+GenericName[sr@ijekavianlatin]=Prikazivač procesa
+GenericName[sr@latin]=Prikazivač procesa
+GenericName[sr]=Приказивач процеса
+GenericName[sv]=Processvisning
+GenericName[tr]=Süreç Görüntüleyici
+GenericName[uk]=Перегляд процесів
+GenericName[zh_CN]=进程查看器
+GenericName[zh_TW]=行程檢視器
 Comment=Show System Processes
 Comment[ca]=Mostra els processos del sistema
 Comment[de]=Systemprozesse anzeigen
@@ -21,45 +49,17 @@ Comment[pt_BR]=Mostra os processos do sistema
 Comment[ru]=Просмотр списка процессов в системе
 Comment[sk]=Zobraziť systémové procesy
 Comment[sl]=Prikaz sistemskih opravil
-Comment[sr]=Приказ системских процеса
 Comment[sr@ijekavian]=Приказ системских процеса
 Comment[sr@ijekavianlatin]=Prikaz sistemskih procesa
 Comment[sr@latin]=Prikaz sistemskih procesa
+Comment[sr]=Приказ системских процеса
 Comment[sv]=Visa systemprocesser
 Comment[tr]=Sistem Süreçlerini Göster
 Comment[uk]=Перегляд системних процесів
 Comment[zh_CN]=显示系统进程
 Comment[zh_TW]=顯示系統行程
-Terminal=true
-Exec=htop
 Icon=htop
-Categories=ConsoleOnly;System;
-GenericName=Process Viewer
-GenericName[ca]=Visor de processos
-GenericName[de]=Prozessanzeige
-GenericName[en_GB]=Process Viewer
-GenericName[es]=Visor de procesos
-GenericName[fi]=Prosessikatselin
-GenericName[fr]=Visualiseur de processus
-GenericName[gl]=Visor de procesos
-GenericName[it]=Visore dei processi
-GenericName[ko]=프로세스 뷰어
-GenericName[nb]=Prosessviser
-GenericName[nl]=Viewer van processen
-GenericName[nn]=Prosessvisar
-GenericName[pl]=Przeglądarka procesów
-GenericName[pt]=Visualizador de Processos
-GenericName[pt_BR]=Visualizador de processos
-GenericName[ru]=Монитор процессов
-GenericName[sk]=Prehliadač procesov
-GenericName[sl]=Pregledovalnik opravil
-GenericName[sr]=Приказивач процеса
-GenericName[sr@ijekavian]=Приказивач процеса
-GenericName[sr@ijekavianlatin]=Prikazivač procesa
-GenericName[sr@latin]=Prikazivač procesa
-GenericName[sv]=Processvisning
-GenericName[tr]=Süreç Görüntüleyici
-GenericName[uk]=Перегляд процесів
-GenericName[zh_CN]=进程查看器
-GenericName[zh_TW]=行程檢視器
+Exec=htop
+Terminal=true
+Categories=System;Monitor;ConsoleOnly;
 Keywords=system;process;task
